### PR TITLE
About ready to deliver

### PR DESCRIPTION
**Pull request by p-funky**
This pull request updates the project to a finished product stage pending review from senior developers

### CODE REVIEW

**Comment by enahomurphy** [comment]
- Structure your folders properly
- Write test to cover variety of edge cases
- Ensure to test inverted index methods in units
- Ensure to implement multiple search query in the inverted index class

File: inverted-index.js [hunk 1]
```diff
@@ -111,15 +111,10 @@ class InvertedIndex {
   /**
    * searchIndex method searches the indexed files for occurences of words
    * @param {String} word - word that one is searching for
-   * @param {Object} filename - specific file to search through
```
Comment (tomily1) [inline inverted-index.js]: line 60 to 62, you might look into `match(/\w+/g)` instead of using replace

File: jasmine-standalone-2.5.2/build/bundle.js [hunk 2]
```diff
@@ -204,6 +204,17 @@ describe('InvertedIndex', () => {
...
+      invIndex.createIndex(valid4, 'valid4.json');
+      expect(invIndex.searchIndex('could', 'valid4.json')).toEqual([ 0, 1 ]);
+      expect(invIndex.searchIndex('testing', 'valid4.json')).toEqual([ 0 ]);
+      expect(invIndex.searchIndex('pass', 'valid4.json')).toEqual([ 1 ]);
+      expect(invIndex.searchIndex('the', 'valid3.json')).toEqual([ 0, 1 ]);
+    });
+  });
 });
 
 },{"../../invalid1.json":1,"../../invalid2.json":2,"../../invalid3.json":3,"../../invalid4.json":4,"../../invalid5.json":5,"../../invalid6.json":6,"../../invalid7.json":7,"../../valid1.json":9,"../../valid2.json":10,"../../valid3.json":11,"../../valid4.json":12}],9:[function(require,module,exports){
```
Comment (tomily1) [inline jasmine-standalone-2.5.2/build/bundle.js]: care to explain why this is here?... if it is a bundled file then I think its meant to be in gitignore

File: inverted-index.js (hunk 1 above)
Comment (p-funky) [reply inverted-index.js]: Seen and implemented. Thanks.

File: jasmine-standalone-2.5.2/build/bundle.js (hunk 2 above)
Comment (p-funky) [reply jasmine-standalone-2.5.2/build/bundle.js]: Yes it is a bundled file but it is needed for travisCI to build.
